.github/workflows/release.yaml

```diff
@@ -0,0 +1,72 @@
+name: release
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  package:
+    name: package and upload release assets
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - linux
+          - darwin
+          - windows
+        arch:
+          - amd64
+          - arm64
+          - arm
+        exclude:
+          - os: darwin
+            arch: arm
+          - os: windows
+            arch: arm64
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
```

Comment:
darwin/arm64 is only supported after golang 1.16

Reply:
ok, fixed. Thx.